package.json

```diff
@@ -54,7 +54,7 @@
     "rollup-plugin-replace": "^2.0.0",
     "rollup-plugin-uglify": "^2.0.1",
     "standard": "^10.0.3",
-    "striptags": "^3.1.1",
+    "striptags": "^2.2.1",
```

Comment:
May I know why we decreased the version of `striptags`?

Reply:
Yeah will be better, if you can add a proper reasoning behind these random changes.